### PR TITLE
Update DateOnly - Add Remarks

### PR DESCRIPTION
**Pull request by michal-ciechan**
## Summary

Adding remarks to DateOnly, mainly to have a link to Format Strings (always found it easier to go through DateTime.ParseExact to get to that page).

### CODE REVIEW

File: xml/System/DateOnly.xml [hunk 1]
```diff
@@ -939,7 +939,15 @@
...
+## Remarks
+ The <xref:System.DateOnly.ParseExact%28System.String%2CSystem.String%29?displayProperty=nameWithType> method parses the string representation of a date, which must be in the format defined by the `format` parameter. It also requires that the \<Date> element of the string representation of a date appear in the order specified by `format`, and that `s` have no white space other than that permitted by `format`.
+ The `format` parameter is a string that contains either a single standard format specifier, or one or more custom format specifiers that define the required format of `s`. For details about valid formatting codes, see [Standard Date and Time Format Strings](/dotnet/standard/base-types/standard-date-and-time-format-strings) or [Custom Date and Time Format Strings](/dotnet/standard/base-types/custom-date-and-time-format-strings).
+> [!NOTE]
+> If `format` is a custom format pattern that does not include date separators (such as "yyyyMMdd"), use the widest form of each custom format specifier. For example, if you want to specify months in the format pattern, specify the wider form, "MM", instead of the narrower form, "M".
```
Comment (michal-ciechan) [inline xml/System/DateOnly.xml:948]: Not sure if this is still the best practice for DateOnly? Assuming it is.

**Comment by dakersnar** [comment]
cc @tarekgh

File: xml/System/DateOnly.xml (hunk 1 above)
Comment (tarekgh) [reply xml/System/DateOnly.xml:948]: Yes, it is still same.

**Review by tarekgh — APPROVED** [review]
Thanks @michal-ciechan

**Pull request merged**